determined-ai/determined
# feat: Add workspace column to global model registry page [WEB-804]

## Description

Adds a filterable and sortable `workspace` column to the Model Registry page. 





## Test Plan

1. Navigate to the `Model Registry` page. 
2. Ensure that a `Workspace` column is shown that shows the workspace avatar.
3. Click the sorting buttons on the workspace column and ensure the table is sorted alphanumerically by workspace name.
4.  Click the filter button on the workspace column and ensure that the table is correctly filtered by the chosen workspaces. 
5. Click the `Clear Filters` button and ensure the filter in (4) is removed. 
 




## Commentary (optional)





## Checklist

~- [ ] User-facing API changes need the "User-facing API Change" label.~
~- [ ] Release notes should be added as a separate file under `docs/release-notes/`.~
  ~See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.~
~- [ ] Licenses should be included for new code which was copied and/or modified from any external code.~

## Ticket

WEB-804
